test/express/middleware-custom-formatters-test.js

```diff
@@ -0,0 +1,29 @@
+const {expect} = require('chai');
+const request = require('supertest');
+
+describe('Simple server - custom formatters', () => {
+    let app;
+    before(() => {
+        return require('./test-simple-server-custom-formatters').then((testServer) => {
+            app = testServer;
+        });
+    });
+    it('should use custom formatter for thrown error', (done) => {
```

Comment:
I suggest to add a test for validating that `middlewareOptions` is being propagated to the custom formatter

Reply:
Sure!

Reply:
Updated existing test to make use of middlewareOptions to avoid having separate server just for that.